### ISSUE TITLE
code review mb706

### ISSUE THREAD

**Issue by mb706** [mlr-org/mlr3hyperband#36]
https://via.hypothes.is/https://github.com/mlr-org/mlr3hyperband/blob/4613f91f64fac2630b58bd56075a356a7e7324ab/R/TunerHyperband.R

https://via.hypothes.is/https://github.com/mlr-org/mlr3hyperband/blob/4613f91f64fac2630b58bd56075a356a7e7324ab/R/nds_selection.R

**Comment by mb706** [comment]
click on the '<' at the upper right if you don't see the comments

**Comment by SebGruber1996** [comment]
> does mlr3hyperband modify the mlr3 logger? If yes, this is bad. If no, this sentence should be "The default logging threshold of the mlr3 logger is warn" and put in parentheses.
How else should I disable info messages of mlr3 calls in the background?
I don't know why, but last time when I didn't set the threshold to "warn", I had cluttered info messages. I did another run today with a new R session without setting the threshold and suddenly the cluttered info logs are gone. I guess I just do, what you said and remove the threshold setting of mlr3 as it is obviously not required. But my first question still holds out of curiosity.
